plugins/gatsby-ample-generator/lib/templates/component/component.js

```diff
@@ -3,13 +3,11 @@ import PropTypes from "prop-types"
 
 import styles from "./styles.module.scss"
 
-const ComponentName = ({ name }) => {
-  return (
-    <div className={styles.component_name}>
-      <p>{name}</p>
-    </div>
-  )
-}
```

Comment:
I'd be happy to have this discussion with you, but I prefer to start with this structure. I know it's an extra couple lines for components that don't need it, but this way it's consistent among every component — using an explicit return.

Reply:
We talked this out in a meeting. I'm fine with the change you've made.